doc/user/sql/create-database.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "CREATE DATABASE"
+description: "`CREATE DATABASE` creates a new database namespace."
+menu:
+  main:
+    parent: 'sql'
+---
+
+`CREATE DATABASE` creates a new database [namespace](../namespaces).
+
+## Conceptual framework
+
+Materialize mimics PostgreSQL's namespace hierarchy, which is:
```

Comment:
I think we should say that we follow the SQL standard's namespace hierarchy, rather than PostgreSQL's.